README.md

```diff
@@ -82,6 +44,80 @@ To find all duplicate files in all repositories run
 
 All repositories must be [hashed](#hash) before.
 
+## Requirements
+
+### Docker
+
+Start containers:
+
+```bash
+docker-compose up -d
+```
+
+Local directories `repositories` and `query` are available as `/repositories` and `/query` inside the container.
+
+Examples:
+
+```bash
+$ docker-compose exec gemini ./hash /repositories
+Hashing 2 repositories in: /repositories
```

Comment:
👍  for detailed example that users could just copy and run.

I would suggest  to remove the output of those commands here and below for 2 reasons:
 - easier to maintain, i.e when format of output is changed - no need to update this
 - consistency, as other examples above do not include it

Reply:
removed